packages/babl/babl-fix-tmpdir-path.patch

```diff
@@ -0,0 +1,53 @@
+diff -uNr babl-0.1.90/babl/babl-cache.c babl-0.1.90.mod/babl/babl-cache.c
+--- babl-0.1.90/babl/babl-cache.c	2022-02-22 05:30:17.000000000 +0800
++++ babl-0.1.90.mod/babl/babl-cache.c	2022-05-17 08:48:49.607069500 +0800
+@@ -29,7 +29,7 @@
+ #ifdef _WIN32
+ #define FALLBACK_CACHE_PATH  "C:/babl-fishes.txt"
+ #else
+-#define FALLBACK_CACHE_PATH  "/tmp/babl-fishes.txt"
++#define FALLBACK_CACHE_PATH  "@TERMUX_PREFIX@/tmp/babl-fishes.txt"
+ #endif
+ 
+ static int
+@@ -154,7 +154,7 @@
+   return dest;
+ }
+ 
+-static int 
++static int
```

Comment:
Please remove these formatting changes.

Reply:
I didn't alter that by myself, it was probably the whitespace trimming on my editor what's causing while making changes to the source code